pegasus/sites.v3/code.org/public/educate/professional-learning/middle-high.md.erb

```diff
@@ -167,14 +186,275 @@ The Code.org Professional Learning Program is open to educators who are interest
+
+</div>
+
+<div class="col-40" style="padding-right: 30px; padding-top: 20px; padding-bottom: 20px;">
+
+<img src="/images/fit-400/marketing/workshop-attendees-2019-middle-high.jpg">
+
+</div>
+
+</div>
+
+<div style="clear: both;"></div>
+<a name="content-and-curriculum"></a>
+
+## Content and curriculum
+Code.org provides a complete CS program combining professional learning and the K-12 curriculum, both of which consistently gets high ratings from teachers.
+
+Middle and high school teachers will be participating in the Code.org professional learning workshop for one of the course below. Teach a different grade level or have fewer hours available with your students? <a href="https://code.org/files/course-pl-options.pdf">Find the right course for your classroom</a>.<br><br>
+
+<div class="col-50">
+
+<center>
+
+<div style="width: 70%;">
+<%=view :display_video_thumbnail, id: "intro_csd", video_code: "uQim0hBHco0", play_button: 'center', letterbox: "false" %>
+
+<h3><a href="https://code.org/educate/csd">CS Discoveries (Grades 6-10)</a></h3>
+
+</center>
+
+[An introductory computer science course](https://code.org/educate/csd) that empowers students to create authentic artifacts and engage with computer science as a medium for creativity, communication, problem solving, and fun.
+
+</div>
+
+</div>
+
+<div class="col-50">
+
+<center>
+<div style="width: 70%;">
+<%=view :display_video_thumbnail, id: "intro_csp", video_code: "DMr1iFYacGQ", play_button: 'center', letterbox: "false" %>
+
+<h3><a href="https://code.org/educate/csp">CS Principles (Grades 9-12)</a></h3>
+
+</center>
+
+[A higher level introductory course](https://code.org/educate/csp) that introduces the foundational concepts of computer science while challenging students to explore how computing and technology can impact the world. Can be taught as an AP or non-AP class
+
+</div>
+
+</div>
+
+<div style="clear: both;">
+
+<table style="border-style:hidden; padding-top: 30px; padding-bottom: 30px; padding-right: 20px; padding-left: 20px;">
+  <tr>
+    <td colspan="100" style="background-color:#00adbc; color:white; text-align:center; font-style:bold; border-style:hidden; font-size:18px;""> Peer reviewed research shows that school participation in the Code.org program causes an estimated <b>5x increase in students that take and pass the AP CS Principles exam</b>.
+      </td>
+    </tr>
+  </table>
+</div>
+
+<div style="clear: both;"></div>
+
+[/optimizely-variant-a]
+
+
+[optimizely-original]
+
 ## <a name="questions"></a>Have questions?
 
+[/optimizely-original]
+
+[optimizely-variant-a]
+
+## Get more information
+
+[/optimizely-variant-a]
+
 <%= view :professional_learning_regional_partner_mini_contact, source_page_id: "middle-high", notes: "Please tell me more about the professional learning program for grades 6-12!" %>
```

Comment:
I've interleaved the original and variant here at the end of the doc because this view, the mini contact form, was unhappy about being included twice on the page, and is itself identical in both cases.